GVRf/Framework/jni/engine/memory/gl_delete.h

```diff
@@ -69,7 +105,19 @@ class GlDelete {
     std::vector<GLuint> vertex_arrays_;
 };
 
-extern GlDelete gl_delete;
+/**
+ * The assumption is threads that do know they are supposed to have
+ * a deleter may only call this method.
+ */
+static GlDelete* getDeleterForThisThread() {
```

Comment:
this static function in header file can increase code size. could be moved to cpp, and make it extern.